### PR TITLE
ENH: Add CI.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add CI for the module:
- Add Circle, Travis and AppVeyor CI configuration `*.yml` files.
- Add the `CTestConfig.cmake` file to submit builds to the **Inisght**
  project in the **open.cdash.org** dashboard.
- Add the Python setup `setup.py` file
- Add the Python wrapping `*.wrap` files.
- Re-use the `README` file documentation in the `itk-module.cmake` file.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@phcerdan @thewtex please have a look at the `*wrap` files, and please report any inconsistencies or bugs. Thanks !

**Comment by thewtex** [comment]
@jhlegarreta thanks for working on these external modules -- it will be awesome to have Python packages for them!

For this module, which has filters for QuadEdgeMesh's, inspire from the wrapping configuration in [`ITKQuadEdgeMeshFiltering`](https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/Filtering/QuadEdgeMeshFiltering/wrapping) -- we cannot use the image wrapping macros.

**Comment by jhlegarreta** [comment]
Thanks for the review. Done in 2b27239. Not sure whether the `*Criterion.wrap` files should be wrapped.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          name: Build and Test
+          no_output_timeout: 1.0h
+          command: |
+            ctest -j 2 -VV -S dashboard.cmake
+  package:
+    working_directory: ~/
+    machine: true
+    steps:
+      - checkout:
+          path: ~/ITKSubdivisionQuadEdgeMeshFilter
```
Comment (thewtex) [inline .circleci/config.yml]: @jhlegarreta This path may be interfering with the execution of `./dockcross-manylinux-download-cache-and-build-module-wheels.sh` and can likely be removed, per

https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/965dcc71ae1179bc254e9e34f12930add4798f4e/%7B%7Bcookiecutter.project_name%7D%7D/.circleci/config.yml#L43

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - run:
+          name: Fetch build script
+          command: |
+            curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
+            chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
+      - run:
+          name: Build Python packages
+          no_output_timeout: 1.0h
+          command: |
+            cd ~/ITKSubdivisionQuadEdgeMeshFilter
```
Comment (thewtex) [inline .circleci/config.yml]: This `cd` could be removed, too.

Comment (jhlegarreta) [reply .circleci/config.yml]: @thewtex 👍 Thanks. Done in 59fb4d4.

File: .circleci/config.yml (hunk 1 above)
Comment (jhlegarreta) [reply .circleci/config.yml]: @thewtex 👍 Thanks. Done in 59fb4d4.

**Comment by jhlegarreta** [comment]
CI is doing its job now :ok_hand: Some issues uncovered

**Pull request merged**